# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/VectorImage.h
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b image/VectorImage.h
--- a/image/VectorImage.h	Wed Aug 21 10:29:14 2019 +0000
+++ b/image/VectorImage.h	Wed Aug 21 12:52:19 2019 +0000
@@ -67,8 +67,6 @@
   void OnSVGDocumentLoaded();
   void OnSVGDocumentError();
 
-  virtual void ReportUseCounters() override;
-
  protected:
   explicit VectorImage(nsIURI* aURI = nullptr);
   virtual ~VectorImage();